src/components/layout.js

```diff
@@ -59,21 +30,51 @@ class Layout extends React.Component {
           padding: `${rhythm(1.5)} ${rhythm(3 / 4)}`,
         }}
       >
-        <header>{header}</header>
+        <header>
+          <p style={{ fontFamily: "var(--ui-font)" }}>
+            {location.pathname === rootPath ? (
+              <a href="https://excalidraw.com">{logo}Open Excalidraw</a>
+            ) : (
+              <>
+                <span style={{ float: "right" }}>
+                  <a href="https://excalidraw.com">Open Excalidraw{logo}</a>
+                </span>
+                <Link to="/archive">Archive</Link>
+              </>
+            )}
+            <span style={{ clear: "both" }} />
+          </p>
+          {location.pathname === rootPath ? (
+            <h1
+              style={{
+                ...scale(1.2),
+                marginTop: 0,
+              }}
+            >
+              {title}
+            </h1>
+          ) : null}
+        </header>
         <main>{children}</main>
         <footer
           style={{
             textAlign: "center",
             padding: `${rhythm(2)} 0`,
+            fontFamily: "var(--ui-font)",
           }}
         >
-          © {new Date().getFullYear()}
-          {" – "}
+          © {new Date().getFullYear()} Excalidraw blog{" "}
+          <a href="https://github.com/excalidraw/excalidraw-blog/graphs/contributors">
+            contributors
+          </a>
+          {" • "}
           <a href="https://twitter.com/excalidraw">Twitter</a>
-          {" – "}
-          <a href="https://github.com/excalidraw">GitHub</a>
-          {" – "}
-          <a href="https://excalidraw.com">Excalidraw</a>
+          {" • "}
+          <a href="https://github.com/excalidraw/excalidraw-blog">
+            Fork this site
```

Comment:
👍🏽

Reply:
It could go either way. I thought it would be useful to list who the copyright actually belongs to, but we don’t need to link to the contributors list. I would like to link to the license, though.